react-training/src/app/screens/Login/index.js

```diff
@@ -2,18 +2,18 @@ import React, { Component } from 'react';
 import { Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
 
-import loginActions from '../../../redux/login/actions';
+import loginActions from '../../../redux/auth/action';
 
 import LoginForm from './LoginForm';
 
 class Login extends Component {
   handleSubmit = values => this.props.requestLogin(values);
 
   render() {
-    const { loggedIn, loggingIn, loginError } = this.props;
+    const { isAuth, loggingIn, authError } = this.props;
 
-    return !loggedIn ? (
-      <LoginForm onSubmit={this.handleSubmit} loginError={loginError} loggingIn={loggingIn} />
+    return !isAuth ? (
+      <LoginForm onSubmit={this.handleSubmit} authError={authError} loggingIn={loggingIn} />
```

Comment:
Here u must render only the `LoginForm`. The redirect logic if is auth must extracted in just one component, like `AuthenticatedRoute`